iothub/device/samples/PnpDeviceSamples/Thermostat/Program.cs

```diff
@@ -0,0 +1,104 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.Text;
+using System.Threading.Tasks;
+using Microsoft.Azure.Devices.Client;
+using Microsoft.Azure.Devices.Shared;
+using PnpHelpers;
+
+namespace TemperatureController
+{
+    public class Program
+    {
+        private const string DeviceConnectionString = "device_connection_string_here";
```

Comment:
How about a command line input to the sample instead? Otherwise there is a chance one might accidentally check this value in.

Reply:
The cross-SDK consensus was to supply this to the sample via environmental variables; so I'll make the change for that.

Reply:
@vinagesh @bikamani : FYI - the env var name to be used for service connection string: IOTHUB_CONNECTION_STRING